mediapipe/graphs/instantmotiontracking/instant_motion_tracking.pbtxt

```diff
@@ -61,28 +61,18 @@ node {
 node {
   calculator: "GlAnimationOverlayCalculator"
   input_stream: "VIDEO:input_video"
-  input_stream: "MODEL_MATRICES:robot_matrices"
-  input_side_packet: "TEXTURE:robot_texture"
-  input_side_packet: "ANIMATION_ASSET:robot_asset_name"
-  output_stream: "robot_rendered_video"
-}
-
-# Renders the final 3d stickers and overlays them on input image.
-node {
-  calculator: "GlAnimationOverlayCalculator"
-  input_stream: "VIDEO:robot_rendered_video"
-  input_stream: "MODEL_MATRICES:dino_matrices"
-  input_side_packet: "TEXTURE:dino_texture"
-  input_side_packet: "ANIMATION_ASSET:dino_asset_name"
-  output_stream: "dino_rendered_video"
+  input_stream: "MODEL_MATRICES:gif_matrices"
+  input_stream: "TEXTURE:gif_texture"
+  input_side_packet: "ANIMATION_ASSET:gif_asset_name"
+  output_stream: "asset_gif_rendered"
 }
 
 # Renders the final 3d stickers and overlays them on input image.
 node {
   calculator: "GlAnimationOverlayCalculator"
-  input_stream: "VIDEO:dino_rendered_video"
-  input_stream: "MODEL_MATRICES:gif_matrices"
-  input_stream: "TEXTURE:gif_texture"
-  input_side_packet: "ANIMATION_ASSET:gif_asset_name"
+  input_stream: "VIDEO:asset_gif_rendered"
+  input_stream: "MODEL_MATRICES:asset_3d_matrices"
+  input_side_packet: "TEXTURE:texture_3d"
+  input_side_packet: "ANIMATION_ASSET:asset_3d"
   output_stream: "output_video"
-}
+}
```

Comment:
super-nit: let's put back the end-of-file newline here, just so it doesn't complain